### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by KarikalStrom** [himashi92/vizviva_brats_2021#1]
Hello, first of all, great job. I am trying to use this model and implement it in an app I'm working on. I can't find the pre-trained model if you didn't share it yet is there anyway you can share it with me. My computer isn't strong enough to run the training.
Thanks in advance.

**Comment by himashi92** [comment]
Hi, you can find the pretrained model here https://drive.google.com/file/d/11YmBPePPmnqE9W40ZqschovmiPx6lZ-2/view?usp=sharing

**Comment by thisisatharva** [comment]
Hey, I was trying to load these pretrained weights using 

`model_maker = getattr(model, 'unet')
model_1 = model_maker.Unet(
        4, 3,
        width=24, norm_layer=get_norm_layer('inorm'), dropout=0.)
model_2 = model.unet.Unet(4, 3, width=32, norm_layer=get_norm_layer('inorm'), dropout=0.)
checkpoint = torch.load('../model_best.pth.tar')
model_1.load_state_dict(checkpoint['state_dict'])`

But I am getting a shape mismatch error. Any reason why this is happening, I followed the code from test.py.
